api/apps/articles/model/save.coffee

```diff
@@ -214,24 +214,13 @@ denormalizedArtworkData = (artwork) ->
     partner:
       name: getPartnerName artwork
       slug: getPartnerLink artwork
-    artist:
```

Comment:
Hmm so removing `artist` now wouldn't be quite right since the front-end depends on there being an `artist` field and will thus break. 

It should come after we've ensured that the front-end is no longer using the field at all -- ie after we've updated Force/MG. Does that make sense?